assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -157,6 +157,7 @@ storage:
             quay.io/kinvolk/bootkube:v0.14.0-helm-amd64 \
             --net=host \
             --dns=host \
+            --hosts-entry=host \
```

Comment:
Have we verified this doesn't have side effects? Also, I believe you're proposing to add this flag to all platforms for the sake of uniformity, however does it make sense to do that on platforms which don't need this functionality? I think it might be better to only add the flag where it's needed, to minimize potential "surprises" in the future.

Reply:
>Have we verified this doesn't have side effects?

I cannot come up with any possible side effects of that beside benefits. This potentially allows to also use CLC snippets for /etc/hosts for names resolving for other platforms too (if using only Terraform code, which I know we don't support).